icu4c/source/i18n/dtfmtsym.cpp

```diff
@@ -1490,6 +1490,27 @@ const char16_t kFormatTagUChar[] = {
 const char16_t kAbbrTagUChar[] = {
     SOLIDUS, LOW_A, LOW_B, LOW_B, LOW_R, LOW_E, LOW_V, LOW_I, LOW_A, LOW_T, LOW_E, LOW_D
 };
+const char16_t kErasTagUChar[] = {
+    LOW_E, LOW_R, LOW_A, LOW_S
+};
+
+struct EraCodeAndName : public UObject {
+    int32_t       eraCode;
+    UnicodeString eraName;
+};
```

Comment:
I'm with Markus on this-- why not just use a string literal, now that that works?  Maybe it's even worth it to convert the other constants.

Reply:
That wasn't the line I was trying to mark!  I meant this comment to apply to `kErasTagUChar` just above this.

Reply:
Yes, will fix this

Reply:
Done